src/main/java/slimeknights/tconstruct/common/data/TConstructBlockLootTables.java

```diff
@@ -177,6 +178,15 @@ private void addSmeltery() {
     this.registerBuildingLootTables(TinkerSmeltery.searedRoad);
     this.registerBuildingLootTables(TinkerSmeltery.searedTile);
     this.registerDropSelfLootTable(TinkerSmeltery.searedGlass.get());
+
+    for (SearedTankBlock.TankType type : SearedTankBlock.TankType.values()) {
+      for (Block block : new Block[] {TinkerSmeltery.searedTank.get(type)}) {
+        this.registerLootTable(block, droppingWithFunctions(block, (builder) -> {
+          return builder.acceptFunction(CopyName.builder(CopyName.Source.BLOCK_ENTITY))
+            .acceptFunction(CopyNbt.builder(CopyNbt.Source.BLOCK_ENTITY).replaceOperation("tank", "tank"));
+        }));
+      }
```

Comment:
Ditch the second for loop, its useless if you have only one entry (the other case I had multiple entries). Just do:
```java
        this.registerLootTable(TinkerSmeltery.searedTank.get(type), (block) -> droppingWithFunctions(block, (builder) -> {
          return builder.acceptFunction(CopyName.builder(CopyName.Source.BLOCK_ENTITY))
            .acceptFunction(CopyNbt.builder(CopyNbt.Source.BLOCK_ENTITY).replaceOperation("tank", "tank"));
        }));
```

Reply:
k.